### DISCUSSION TITLE
Trouble compiling under Windows 10 with VSCode

### DISCUSSION THREAD

**Discussion by jdeline** [grblHAL/RP2040#11] [Q&A]
I'm curious how many people had problems compiling the RP2040 version under VSCode in Windows 10. I have Windows 10 on my desktop computer and I installed VSCode and tried to compile Grbl RP2040, but VSCode kept reporting that it couldn't find a cmake generator. I researched online and tried a few things, but could not get around that error. However, I was able to compile it under Linux with VSCode. Quite easily, actually.

**Comment by andrewmarles** [comment]
You just need to follow the Raspberry Pi procedure for installing the C SDK and starting VSCode with the appropriate environment set up.  But it is not as easy as under linux.

I believe this script can be quite helpful

https://github.com/ndabas/pico-setup-windows
